tslint.json

```diff
@@ -69,6 +70,54 @@
             "check-separator",
             "check-type"
         ],
-        "no-string-literal": true
+
+        // TODO
+        "arrow-parens": false, // [true, "ban-single-arg-parens"]
+        "arrow-return-shorthand": false,
+        "ban-types": false,
+        "callable-types": false,
+        "forin": false,
+        "interface-name": false, // [true, "never-prefix"],
+        "member-access": false, // [true, "no-public"]
+        "no-angle-bracket-type-assertion": false,

```

Comment:
I like these...

Reply:
There is a lint rule mandating that *only* that kind of type assertion may be used, should we use that instead?

Reply:
No; I'm of the opinion that we should only use `as` casts since they're the only style that works in all language variants ;)

Also angle brackets are sharp and pointy.